galite-core/src/main/kotlin/org/kopi/galite/domain/Domain.kt

```diff
@@ -72,12 +74,25 @@ open class Domain<T>(val width: Int? = null,
                                                  width ?: 0,
                                                  min as? Int ?: Int.MIN_VALUE,
                                                  max as? Int ?: Int.MAX_VALUE)
-        String::class -> VStringField(block.buffer,
-                                      width ?: 0,
-                                      height ?: 1,
-                                      visibleHeight ?: 1,
-                                      0,  // TODO
-                                      styled)
+        String::class -> {
+          if(visibleHeight != height) {
+            VStringField(block.buffer,
+                         width ?: 0,
+                         height ?: 1,
+                         visibleHeight ?: 0,
```

Comment:
can you check here if we need to put **?: 0** or **?: 1**

Reply:
It should be 0 if `visibleHeight` is `null`